src/js/components/FormField/stories/CustomBorder.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Box, FormField, CheckBox, Grommet } from 'grommet';
+
+const FormFieldCustomBorder = props => {
+  const [checkBox1, setCheckBox1] = useState(false);
+  const [checkBox2, setCheckBox2] = useState(false);
+  const [checkBox3, setCheckBox3] = useState(false);
+
+  return (
+    <Grommet theme={grommet}>
+      <Box justify="center" pad="large" direction="row" gap="medium">
+        <FormField label="Default border" htmlFor="check-box" {...props}>
+          <Box pad={{ horizontal: 'small', vertical: 'xsmall' }}>
```

Comment:
Since `pad={{ horizontal: 'small', vertical: 'xsmall' }}` is repeated three times, we can extract it to be a variable.